docs/guided-remediation.md

```diff
@@ -0,0 +1,266 @@
+---
+layout: page
+title: Guided Remediation
+permalink: /experimental/guided-remediation/
+parent: Experimental Features
+nav_order: 3
+---
+
+# Guided Remediation
+
+Experimental
+{: .label }
+
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+Guided remediation aims to help developers with fixing with the high number of known vulnerabilities in dependencies typically reported by vulnerability scanners by providing a small number of actionable steps.
+
+The `osv-scanner fix` subcommand leverages [deps.dev](https://deps.dev) to provide automated and guided remediation of vulnerabilities in your project's dependencies by suggesting upgrades to dependencies.
```

Comment:
This now feels a bit repetitive. How about folding this into the previous paragraph, and moving the deps.dev mention to the resolution point below?

```
Guided remediation (`osv-scanner fix`) aims to help developers with ....
...

...
- Resolution and analysis of the entire transitive graph (leveraging deps.dev)...
```

Reply:
Done